src/git_function.jl

```diff
@@ -67,6 +68,19 @@ function git(; adjust_PATH::Bool = true, adjust_LIBPATH::Bool = true)
         git_cmd = addenv(git_cmd, "PATH" => join(path, pathsep), LIBPATH_env => join(libpath, pathsep))
     end
 
+    # Add git-lfs
+    if Git_LFS_jll.is_available()
+        # Read path from git_cmd.env as it can be modified above
+        idx = findfirst(startswith("PATH="), git_cmd.env)
+        path = if isnothing(idx)
+            ""
+        else
+            # dropping the `PATH=` part
+            git_cmd.env[idx][6:end]
+        end
+        path = vcat(dirname(Git_LFS_jll.git_lfs_path), path)
+        git_cmd = addenv(git_cmd, "PATH" => join(path, pathsep))::Cmd
+    end
```

Comment:
Why not modifying this above where you still have access to the `path` variable (which is an array of strings), instead of fiddling with `startswith`/`findfirst` and such?

Reply:
If we did it above I think we would have to add code into the `if` statements that do different things based on the current platform. So we would duplicate the same code (adding `git-lfs` to the path) in multiple places which I think is less clear than doing it in one place at the end.